src/pages/MainPage.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Navigation from '../components/Navigation';
+import AllArticles from './AllArticles';
+
+
+function MainPage() {
+    return (
+      <>
+        <header>
+          <Navigation />
+          <h1>Welcome to NC News</h1>
+        </header>
+        <main>
+          <AllArticles />
+        </main>
+      </>
+    );
+}
```

Comment:
This seems like a strange behaviour. Having a page that displays a component as a child and then having a different page displaying that single component on its own? I would recommend either removing all articles here or removing the path to /articles as it doesnt seem like a good user experience to see the same thing in two slightly different formats in two different endpoints.